Auths finish before testing

CyberPanda263 opened the pull request:
Developed authentication and documentation on Swager

CyberPanda263 committed:
Auths finish before testing

Developed authentication and documentation on Swager

Spinkll commented on src/db/User.js:
```diff
@@ -0,0 +1,26 @@
+import {model, Schema } from "mongoose";
+
+const usersSchema = new Schema(
+    {
+        photo: {
+            type: String,
+        },
```
мені здається тут потрібно додати дефолт

CyberPanda263 committed:
Improvement of PATCH

Now you don't need to send the user id, you can only change the logged-in user

Spinkll commented on src/db/Session.js:
```diff
@@ -0,0 +1,31 @@
+            type: String,
+            require: true
+
+        },
```
`userId: { type: Schema.Types.ObjectId, ref: 'users' }`

CyberPanda263 committed:
fixes user photo

CyberPanda263 committed:
fixes for theme

Now the theme will be saved to the user, depending on the theme, the default image will change
